geomesa-plugin/src/main/scala/org/locationtech/geomesa/plugin/wms/CoverageReader.scala

```diff
@@ -71,11 +72,11 @@ class CoverageReader(val url: String) extends AbstractGridCoverage2DReader() wit
     case (columnFamily, columnQualifier) => new Text("~" + columnFamily + "~" + columnQualifier)
   }
 
-  this.crs = AbstractGridFormat.getDefaultCRS
-  this.originalEnvelope = new GeneralEnvelope(Array(-180.0, -90.0), Array(180.0, 90.0))
-  this.originalEnvelope.setCoordinateReferenceSystem(this.crs)
-  this.originalGridRange = new GridEnvelope2D(new Rectangle(0, 0, 1024, 512))
-  this.coverageFactory = CoverageFactoryFinder.getGridCoverageFactory(this.hints)
+  val crs = CRS.decode("EPSG:4326")
+  val originalEnvelope = new GeneralEnvelope(Array(-180.0, -90.0), Array(180.0, 90.0))
+  originalEnvelope.setCoordinateReferenceSystem(crs)
+  val originalGridRange = new GridEnvelope2D(new Rectangle(0, 0, 1024, 512))
+  val coverageFactory = CoverageFactoryFinder.getGridCoverageFactory(hints)
```

Comment:
thats strange, yes it builds fine for me
i wonder why that is
I don't think they are overriding anything